CommandLine.c

```diff
@@ -384,8 +386,10 @@ int CommandLine_run(const char* name, int argc, char** argv) {
    if (flags.pidMatchList)
       Hashtable_delete(flags.pidMatchList);
 
-   /* Delete Settings last, since it can get accessed in the crash handler */
+   /* Delete these last, since they can get accessed in the crash handler */
    Settings_delete(settings);
+   Hashtable_delete(dc);
+   Hashtable_delete(dm);
```

Comment:
These two calls lack a NULL check. Adding the check should resolve the compile issue.

An alternative would be to always initialize this Hashtable instead of using `NULL` when a platform does not support dynamic columns.